doc/connectivity/networking/api/lwm2m.rst

```diff
@@ -413,16 +413,16 @@ For a more detailed LwM2M client sample see: :ref:`lwm2m-client-sample`.
 
 Multi-thread usage
 ******************
-Writing a value to a resource can be done using functions like lwm2m_engine_set_u8. When writing
+Writing a value to a resource can be done using functions like lwm2m_set_u8. When writing
 to multiple resources, the function lwm2m_registry_lock will ensure that the
 client halts until all writing operations are finished:
 
 .. code-block:: c
 
   lwm2m_registry_lock();
-  lwm2m_engine_set_u32("1/0/1", 60);
-  lwm2m_engine_set_u8("5/0/3", 0);
-  lwm2m_engine_set_float("3303/0/5700", &value);
+  lwm2m_set_u32(&LWM2M_OBJ(1, 0, 1), 60);
+  lwm2m_set_u8(&LWM2M_OBJ(5, 0, 3), 0);
+  lwm2m_set_float(&LWM2M_OBJ(3303, 0, 5700), &value);
```

Comment:
This should be `lwm2m_set_f64` now, right?

Reply:
Done